### PR TITLE
Added Modern Links

### PR DESCRIPTION
**Pull request by Magicode1**
I went through and updated the old links (Pocketmine.net mostly) and updated them.

Let me know if I missed anything.

### CODE REVIEW

File: source/plugins.rst [hunk 1]
```diff
@@ -83,7 +83,7 @@ Create .phar files
 ------------------
 
 The easiest way to release a plugin is in .phar format.
-To create a .phar you need `DevTools <https://github.com/PocketMine/DevTools/releases>`_.
+To create a .phar you need `DevTools <https://poggit.pmmp.io/p/DevTools/68>`_.
```
Comment (dktapps) [inline source/plugins.rst]: Prefer using the GitHub link.

**Comment by dktapps** [comment]
... and please don't use the web editor 😞

**Comment by Magicode1** [comment]
Ok, I will make the changes. (I will make the changes locally then, not using web editor 👍 ) I was at school when I was editing, so that why I wasn't using notepad or something.

**Comment by dktapps** [comment]
The web editor is fine for small edits, but I discourage its use. Working locally is much better.

**Pull request merged**